src/main/java/io/github/oliviercailloux/jconfs/calendar/CalendarOnline.java

```diff
@@ -89,13 +89,18 @@ public CalendarOnline(CalDavCalendarGeneric connector) {
 		CalendarQuery calendarQuery = searchQuery.generate();
 		Set<Conference> listConferencesUser = new LinkedHashSet<>();
 		List<Calendar> calendarsResult = connector.collectionCalendarsOnline.queryCalendars(this.connector.httpclient, calendarQuery);
+		
 		for (Calendar calendar : calendarsResult) {
+			try { 
 			ComponentList<VEvent> componentList = calendar.getComponents(Component.VEVENT);
 			Iterator<VEvent> eventIterator = componentList.iterator();
 			while (eventIterator.hasNext()) {
 				VEvent vEventFound = eventIterator.next();
 				listConferencesUser.add(ConferenceReader.createConference(vEventFound));
 			}
+			} catch(Exception e) {
+				System.out.println("Conference unreadable");
+			}
```

Comment:
interdiction to use try{
                          
                                 }case ()
 use Exception